src/model/schema.js

```diff
@@ -475,6 +470,70 @@ export default class Schema {
 		}, { priority: 'high' } );
 	}
 
+	/**
+	 * Registers custom properties to a given attribute.
```

Comment:
Sets? Registering means saying "I exist!". We register nodes (`Schema#register()`). You can call `register( foo )` only once too (cause you report your existence once).

Reply:
Or rather:

> Defines properties of the given attribute. 
>
> This method allows assigning additional information to model attributes. Currently, it is only used to mark formatting attributes (like `bold`, `italic`) which allows differentiating them from semantic attributes (such as `src`, `listType`, etc.). This knowledge is then used by features like "Remove format".

And that's it. Good documentation should back all big words by giving real-life examples.

Reply:
I see you used "metadata" below. That's a nice word too. You may try to use it here too. It may be better than "information" or "properties".

Reply:
The introduction was modified to shortly mention cases and link `AttributeProperties#isFormatting` example for further reading.

Unfortunately I could not link to isFormatting property directly due to ckeditor/ckeditor5-dev#504.

PTAL